### ISSUE TITLE
Investigat wabac.js 2.17.0 changes

### ISSUE THREAD

**Issue by benoit74** [openzim/warc2zim#190]
There is a new wabac.js 2.17.0 release ([changelog here](https://github.com/webrecorder/wabac.js/compare/v2.16.13...v2.17.0))

No change in the file we need to keep in sync (`src/rewrite/dsruleset.js`) but interesting new test cases around HTML/JS (body onload, background).

To be investigated further

**Comment by benoit74** [comment]
There is now a 2.17.1, maybe interesting as well

**Comment by kelson42** [comment]
@benoit74 Is this ticket still valid, seems important to me we stick to latest version of wabac. Maybe there is another issue to handle this problematic?

**Comment by benoit74** [comment]
Yes it is ; it is just a complex thing since code has to be analyzed manually ; as mentioned, no impact expected on rules sets, more interesting test cases to check if we already cover them as well or not.

**Comment by benoit74** [comment]
So there was only two interesting cases in the test set of 2.17.1:
- `onload` property of a `<body>` in HTML => already handled 
- `background` property of a `<td>` in HTML => I don't know why one wants to support this, at least it does not look like a standard at all, not even a deprecated attribute, see e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td

Other than that, the process to follow-up on fuzzy-rules and DS rules which we reimplemented from wabac.js is in place, with a reminder in the release issue and the details in respective files that have to be updated.

![image](https://github.com/openzim/warc2zim/assets/7102089/0dad6431-417e-4181-840d-510fd057dd65)

https://github.com/openzim/warc2zim/blob/main/rules/rules.yaml

https://github.com/openzim/warc2zim/blob/main/src/warc2zim/content_rewriting/ds.py

Nota: DS rewriting will probably get dropped with https://github.com/openzim/warc2zim/issues/328 anyway
